### DISCUSSION TITLE
How to add transparency

### DISCUSSION THREAD

**Discussion by Aster75** [cefsharp/CefSharp#3717] [General]
browser.BackColor = Color.FromArgb(50, 100, 4, 10);
this method is not supported

**Comment by amaitland** [comment]
Are you using WinForms or WPF? Only WPF supports transparency.

**Reply by Aster75** [reply]
WinForms

**Reply by amaitland** [reply]
There is currently no support for transparency in WinForms. Upstream support needs to be added first.

Potentially if https://bitbucket.org/chromiumembedded/cef/pull-requests/267 is merged we can add support. It is not clear if/when this will be merged.
